packages/preferences/package.json

```diff
@@ -0,0 +1,44 @@
+{
+	"name": "@wordpress/preferences",
+	"version": "1.0.0-prerelease",
+	"private": true,
```

Comment:
Other public packages depend on this package so it shouldn't be private. See https://github.com/WordPress/gutenberg/pull/39390.